pype/plugins/ftrack/publish/collect_ftrack_family.py

```diff
@@ -0,0 +1,41 @@
+"""
+Requires:
+    none
+
+Provides:
+    instance     -> families ([])
+"""
+import re
+import pyblish.api
+
+
+class CollectFtrackFamily(pyblish.api.InstancePlugin):
+    """
+        Adds explicitly 'ftrack' to families to upload instance to FTrack.
+
+        Uses selection by combination of hosts/families/tasks names (regex)
+    """
+    label = "Collect FTrack Family"
+    order = pyblish.api.CollectorOrder + 0.4999
```

Comment:
Collector should be processed before `CollectFtrackApi` plugin. Now it does not matter much but `CollectFtrackApi` could check if make sence to create ftrack session.

Reply:
This collector should happen before `CollectFtrackApi`.

Reply:
Oh, I misunderstood for the first time.